lib/daru/dataframe.rb

```diff
@@ -2015,6 +2015,22 @@ def split_by_category cat_name
       end
     end
 
+    # returns array of row tuples at given index(s)
+    def access_row_tuples_by_indexs *indexes
+      positions = @index.pos(*indexes)
+
+      return populate_row_for(positions) if positions.is_a? Numeric
+
+      res = []
+      new_rows = @data.map { |vec| vec[*indexes] }
+      indexes.each do |index|
+        tuples = []
+        new_rows.map { |row| tuples += [row[index]] }
+        res << tuples
+      end
+      res
+    end
```

Comment:
[I wanted to get the specific rows of the dataframe. ](https://github.com/SciRuby/daru/pull/330/files#r111128866)

Example : 

```
irb(main):005:0> df
=> #<Daru::DataFrame(5x3)>
       a   b   c
   0   1  11  11
   1   2  12  22
   2   3  13  33
   3   4  14  44
   4   5  15  55
irb(main):006:0> df.access_row_tuples_by_indexs 0,1,4
=> [[1, 11, 11], [2, 12, 22], [5, 15, 55]]
```